Analysis/HiggsTauTau/filelists/Oct02_MC_80X_reHLT_VBFHToTauTau_M-125.dat

```diff
@@ -0,0 +1,51 @@
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_39.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_3.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_45.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_24.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_17.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_50.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_49.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_19.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_43.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_32.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_37.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_48.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_27.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_10.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_33.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_44.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_14.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_51.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_35.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_41.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_29.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_12.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_40.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_31.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_34.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_15.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_11.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_8.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_30.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_23.root
+VBFHToTauTau_M125_13TeV_powheg_pythia8/crab_VBFHToTauTau/161002_110632/0000/EventTree_4.root
```

Comment:
So just on these, I know there's only two relatively short filelists here so it's not really important, but could be worth putting them in a tarball anyway in the future. But up to you :-)